.gitignore

```diff
@@ -1,2 +1 @@
 node_modules
-lib
```

Comment:
Is there a particular reason why lib is commented out in the gitignore instead of perhaps npmignore? On the flipside, unprocessed sources can be npm ignored